examples/pligos/configs/golang-hello-world/pligos.yaml

```diff
@@ -1,16 +1,51 @@
-version: "1"
+kind: pligos
```

Comment:
Also here again with `kind`, maybe make it `pligos:` and put whatever needed inside.